qa/sc_evm_sync_status_after_consensus_fork.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python3
+import logging
+import time
+from datetime import datetime
+
+from SidechainTestFramework.sc_boostrap_info import SCNodeConfiguration, SCCreationInfo, MCConnectionInfo, \
+    SCNetworkConfiguration
+from SidechainTestFramework.sc_test_framework import SidechainTestFramework
+from SidechainTestFramework.scutil import bootstrap_sidechain_nodes, start_sc_nodes, generate_next_blocks, \
+    connect_sc_nodes, stop_sc_node, start_sc_node, wait_for_sc_node_initialization, \
+    generate_next_block, EVM_APP_BINARY, AccountModel, disconnect_sc_nodes_bi
+from test_framework.util import assert_equal, assert_true, fail, initialize_chain_clean, start_nodes, \
+    websocket_port_by_mc_node_index
+from httpCalls.block.forgingInfo import http_block_forging_info
+
+
+"""
+Test that the eth_syncing rpc method answer in a correct way if the node is currently syncing a batch of new blocks
+from another peer
+
+Configuration:
+    Start 1 MC nodes and 2 SC nodes.
+
+Test:
+    - Test that node can detect `isSync` after being restarted when the ActiveSlotCoefficientFork has been activated.
+"""
+
+WITHDRAWAL_EPOCH_LENGTH = 10
+
+class EvmSyncStatus(SidechainTestFramework):
+    number_of_mc_nodes = 1
+    number_of_sidechain_nodes = 2
+
+    def setup_chain(self):
+        initialize_chain_clean(self.options.tmpdir, self.number_of_mc_nodes)
+
+    def setup_network(self, split=False):
+        self.nodes = self.setup_nodes()
+        self.sync_all()
+
+    def setup_nodes(self):
+        # start MC node
+        return start_nodes(self.number_of_mc_nodes, self.options.tmpdir,
```

Comment:
Consider adding `extra_args=self.debug_extra_args` in case a user wants to run the debugger via `--debugnode=0` option